tests/src/system/basic/WskBasicTests.scala

```diff
@@ -220,15 +220,14 @@ class WskBasicTests
             val params = Map("a" -> "A".toJson)
             assetHelper.withCleaner(wsk.action, name) {
                 (action, _) =>
-                    action.create(name, file, parameters = params, shared = Some(true))
+                    action.create(name, file, parameters = params)
                     action.create(name, None, parameters = Map("b" -> "B".toJson), update = true)
             }
             val stdout = wsk.action.get(name).stdout
             stdout should not include regex(""""key": "a"""")
             stdout should not include regex(""""value": "A"""")
             stdout should include regex (""""key": "b""")
             stdout should include regex (""""value": "B"""")
-            stdout should include regex (""""publish": true""")
```

Comment:
keep this but change true to false (because we want to show if an entity is private or public - it'll just be done by the controller based on the package visibility). (Apply to all the of the below.)